packaging/preremove.sh

```diff
@@ -43,124 +40,122 @@ fi
-#echo "IRODS_HOME_DIR=[$IRODS_HOME]"
+#echo "IRODS_HOME=[$IRODS_HOME]"
 #echo "SERVER_TYPE=[$SERVER_TYPE]"
 #echo "IRODS_SERVICE_ACCOUNT_NAME=[$IRODS_SERVICE_ACCOUNT_NAME]"
 #echo "IRODS_SERVICE_GROUP_NAME=[$IRODS_SERVICE_GROUP_NAME]"
 
 # determine whether this is an upgrade
 if [ "$PACKAGER_COMMAND" -eq "$PACKAGER_COMMAND" ] 2>/dev/null ; then
-  # integer, therefore rpm
-  if [ $PACKAGER_COMMAND -gt 0 ] ; then
-    PACKAGEUPGRADE="true"
-  else
-    PACKAGEUPGRADE="false"
-  fi
+    # integer, therefore rpm
+    if [ $PACKAGER_COMMAND -gt 0 ] ; then
+        PACKAGEUPGRADE="true"
+    else
+        PACKAGEUPGRADE="false"
+    fi
 else
-  # string, therefore deb
-  if [ "$PACKAGER_COMMAND" = "upgrade" ] ; then
-    PACKAGEUPGRADE="true"
-  else
-    PACKAGEUPGRADE="false"
-  fi
+    # string, therefore deb
+    if [ "$PACKAGER_COMMAND" = "upgrade" ] ; then
+        PACKAGEUPGRADE="true"
+    else
+        PACKAGEUPGRADE="false"
+    fi
 fi
 
 #echo "PACKAGEUPGRADE=[$PACKAGEUPGRADE]"
 
 if [ "$PACKAGEUPGRADE" == "false" ] ; then
-	# =-=-=-=-=-=-=-
-	# determine if we can remove the resource from the zone
-	if [ "$SERVER_TYPE" == "resource" ] ; then
-
-	    hn=`hostname`
-	    dn=`hostname -d`
-	    fhn="$hn.$dn"
-	    echo "Testing Safe Resource Removal"
-	    do_not_remove="FALSE"
-
-	    # =-=-=-=-=-=-=-
-	    # do a dryrun on the resource removal to determine if this resource server can
-	    # be safely removed without harming any data
+    # =-=-=-=-=-=-=-
+    # determine if we can remove the resource from the zone
+    if [ "$SERVER_TYPE" == "resource" ] ; then
+
+        hn=`hostname`
+        dn=`hostname -d`
+        fhn="$hn.$dn"
+        echo "Testing Safe Resource Removal"
+        do_not_remove="FALSE"
+
+        # =-=-=-=-=-=-=-
+        # do a dryrun on the resource removal to determine if this resource server can
+        # be safely removed without harming any data
             resources_to_remove=()
-	    for resc in `su -c "iadmin lr" $IRODS_SERVICE_ACCOUNT_NAME`
-	    do
-		# =-=-=-=-=-=-=-
-		# for each resource determine its location.  if it is this server then dryrun
-		loc=$( su -c "iadmin lr $resc | grep resc_net | cut -d' ' -f2" $IRODS_SERVICE_ACCOUNT_NAME )
-
-		if [[ $loc == $hn || $loc == $fhn ]]; then
-			rem=$( su -c "iadmin rmresc --dryrun $resc | grep SUCCESS" $IRODS_SERVICE_ACCOUNT_NAME )
-			if [[ "x$rem" == "x" ]]; then
-				# =-=-=-=-=-=-=-
-                                # dryrun for a local resource was a failure, set a flag
-                                do_not_remove="TRUE"
-                        else
-                                # =-=-=-=-=-=-=-
-				# dryrun for a local resource was a success, add resc to array for removal
-                                echo "  Adding [$resc] for removal..."
-				resources_to_remove+=($resc)
-			fi
-		fi
-	    done
-
-	    if [[ "$do_not_remove" == "TRUE" ]]; then
+        for resc in `su -c "iadmin lr" $IRODS_SERVICE_ACCOUNT_NAME`; do
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use $(..) instead of legacy `..`.](https://app.codacy.com/app/trel/irods/pullRequest?prid=4593032)